ubi8-rust-builder/Dockerfile

```diff
@@ -19,6 +19,8 @@ RUN microdnf update --disablerepo=* --enablerepo=ubi-8-appstream-rpms --enablere
   && microdnf install --disablerepo=* --enablerepo=ubi-8-appstream-rpms --enablerepo=ubi-8-baseos-rpms curl findutils gcc gcc-c++ make cmake openssl-devel pkg-config systemd-devel unzip clang krb5-libs libkadm5 -y \
   && rm -rf /var/cache/yum
 
+# Container Storage Interface is defined using GRPC/Protobuf, our operators that use it (secret-operator/listener-operator) require
+# protoc via Prost (https://github.com/tokio-rs/prost).
```

Comment:
Could you add information on how we can find out which version we need?

Reply:
5d7c64f0fff4c0c91f5ff02d83c7a61369cb32c7